de.fu_berlin.inf.dpp.stf.test/test/eclipse/de/fu_berlin/inf/dpp/stf/test/consistency/EditDuringInvitationStressTest.java

```diff
@@ -17,7 +17,7 @@
 import de.fu_berlin.inf.dpp.stf.client.tester.SarosTester;
 import de.fu_berlin.inf.dpp.stf.client.util.Util;
 import de.fu_berlin.inf.dpp.stf.shared.Constants.TypeOfCreateProject;
-import de.fu_berlin.inf.dpp.stf.test.Constants;
+import de.fu_berlin.inf.dpp.stf.test.stf.Constants;
```

Comment:
Change of the package is correct because the Constants are now in the other stf project.